daos-cortx/docs/datamovment_with_s3.md

```diff
@@ -0,0 +1,139 @@
+# Daos object movement using aws s3cli
+
+- Daos source container to Daos destination container object movement is possible using aws s3 cli. This documents is the setup guide to perform mentioned exercise.
```

Comment:
Document describes data movement between DAOS and CORTX using Posix Interface on DAOS and S3 interface on CORTX. This demonstrates two workflows using AWS S3 cli tool.
1. DAOS to CORTX (posix to S3)
2. CORTX to DAOS (S3 to posix)

Reply:
done